src/ui/components/DismissibleTextForm/styles.scss

```diff
@@ -1,30 +1,58 @@
+@import "~photon-colors/colors";
+@import "~amo/css/inc/vars";
+@import "~core/css/inc/mixins";
+@import "~core/css/inc/vars";
 @import "~ui/css/vars";
 
+.DismissibleTextForm-form {
+  margin: 6px 0;
+
+  @include respond-to(medium) {
+    margin: 12px 0;
+  }
+}
+
 .DismissibleTextForm-buttons {
   display: flex;
   justify-content: space-between;
+  margin-top: 6px;
+
+  @include respond-to(medium) {
+    margin-top: 24px;
+
+    .Button {
+      padding: 0 64px;
```

Comment:
I see now this is for making the button bigger.

I think this should be a button option (something like `Button--large` similar to our existing `Button--small`). I know it's a pain but I have seen a lot of code bases with tiny variances on buttons because they are modified in a non-central place.

It's fine to do it as a follow-up bug if it's a pain here, but we should at least have it filed (and maybe linked to #3398) to have a large button style any component can use rather than individual components changing button styles. It's fine to align/position buttons here, but I think anything we do *inside* the button should be in the button style as an option.

Reply:
Good call, I addressed it. I was hesitating before only because I didn't know if we wanted to apply the style globally or not. Using a new button style is a safe approach because it won't be applied to any existing buttons.